tests/components/rainbird/test_switch.py

```diff
@@ -214,32 +213,26 @@ async def test_irrigation_service(
     ("yaml_config", "config_entry_data"),
     [
         (
+            {},
             {
-                DOMAIN: {
-                    "host": HOST,
-                    "password": PASSWORD,
-                    "trigger_time": 360,
-                    "zones": {
-                        1: {
-                            "friendly_name": "Garden Sprinkler",
-                        },
-                        2: {
-                            "friendly_name": "Back Yard",
-                        },
-                    },
-                }
+                "host": HOST,
+                "password": PASSWORD,
+                "trigger_time": 360,
+                "serial_number": "0x1263613994342",
+                "imported_names": {
+                    "1": "Garden Sprinkler",
+                    "2": "Back Yard",
+                },
             },
-            None,
         )
     ],
 )
-async def test_yaml_config(
+async def test_yaml_imported_config(
     hass: HomeAssistant,
     setup_integration: ComponentSetup,
     responses: list[AiohttpClientMockResponse],
 ) -> None:
     """Test switch platform with fake data that creates 7 zones with one enabled."""
```

Comment:
While you're here, i see this existing comment was incorrect, so lets refresh with the new test behavior:

```suggestion
    """Test a config entry that was previously imported from yaml."""
```